CPP/GameAPI/RSDKv5/EngineAPI.hpp

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include "Types.hpp"
+#include "RSDKv5/Graphics/Graphics.hpp"
```

Comment:
Be careful with this. I'm sure this would work fine, but we use relative paths everywhere else to be absolutely sure.